src/hotspot/share/compiler/compilerThread.hpp

```diff
@@ -38,10 +45,25 @@ class CompileQueue;
 class CompilerCounters;
 class IdealGraphPrinter;
 
+class CompilerThreadTimeoutGeneric : public CHeapObj<mtCompiler> {
+ public:
+  CompilerThreadTimeoutGeneric() {};
+  void arm() { return; };
+  void disarm() { return; };
```

Comment:
You can remove the `return`:
```suggestion
  void arm() {};
  void disarm() {};
```